Autopilot/Path Manager/MPL3115A2.c

```diff
@@ -1,22 +1,34 @@
 /*
  * File:   MPL3115A2.c
- * Author: Chris Hajduk
+ * Author: Chris Hajduk & Rosa Chen
  *
  * Created on April 5, 2014, 7:08 PM
  */
+#include <stdio.h>
+#include <stdint.h>
+#include <stdlib.h>
 #include "MPL3115A2.h"
 #include "main.h"
 #include "../Common/Utilities/Logger.h"
+#include "../AttitudeManager/delay.h"
```

Comment:
These delay functions are old and inaccurate, and were removed for a reason. Better delay functions are available by including xc.h (which should be included in main.h).